api/v1alpha1/kubernetes_helpers.go

```diff
@@ -9,7 +9,7 @@ import (
 	appv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
-	"k8s.io/utils/pointer"
```

Comment:
can you also update golanglint-ci setting for this package?

Reply:
https://github.com/envoyproxy/gateway/blob/f4252e6685a4378a9b2e75f802507711c464dfa6/tools/linter/golangci-lint/.golangci.yml#L27

Reply:
Updated.